gradlew

```diff
@@ -30,6 +48,7 @@ die ( ) {
 cygwin=false
 msys=false
 darwin=false
+nonstop=false
```

Comment:
Maybe add a comment about what "nonstop" is. Or maybe this is something well-known?